src/Symfony/Component/Form/Extension/HttpFoundation/EventListener/BindRequestListener.php

```diff
@@ -42,9 +42,6 @@ public function preBind(FormEvent $event)
             return;
         }
 
-        // Uncomment this as soon as the deprecation note should be shown
-        // trigger_error('Passing a Request instance to Form::submit() is deprecated since version 2.3 and will be disabled in 3.0. Call Form::process($request) instead.', E_USER_DEPRECATED);
```

Comment:
This will fix that the comment is not removed in 2.7: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Form/Extension/HttpFoundation/EventListener/BindRequestListener.php#L48